nix/pkgs.nix

```diff
@@ -0,0 +1,12 @@
+# our packages overlay
```

Comment:
Can we give this file a better name? `plutusHaskell.nix`?

Reply:
again, this is within the naming convetion of what I've found in other repos.

Reply:
y tho

Reply:
I got rid of this.